backend/experiment/tests/test_admin_experiment.py

```diff
@@ -228,7 +228,7 @@ def setUp(self):
     def test_related_experiment_with_experiment(self):
```

Comment:
Should the name of this test be adjusted? `related_experiment_with_experiment` is not clear, I'm guessing we're testing blocks & phases here?